tests/unit/sagemaker/tensorflow/test_estimator.py

```diff
@@ -143,6 +143,7 @@ def _create_train_job(tf_version, horovod=False, ps=False, py_version="py2", smd
             }
         ],
         "profiler_config": {
+            "DisableProfiler": False,
```

Comment:
Why do you need to pass this parameter?

Reply:
This argument is later on used in validation in all the related test. If I don't specify DisableProfiler:False here, later on the validation check will throw error.